src/mdast.rs

```diff
@@ -447,6 +450,7 @@ pub enum AttributeContent {
 
 /// MDX: attribute value.
 #[derive(Clone, Debug, Eq, PartialEq)]
+#[non_exhaustive]
 pub enum AttributeValue {
```

Comment:
I think these are supposed to be exhaustive?

Reply:
For AttributeValue and AttributeContent, I'll trust you, as I don't even know what they're about as I never used JSX markdown ^^'

For Node, I'm pretty sure keeping it non-exhaustive would be better: it seems likely to me that some future evolution will require adding node types given the length of the enum (eg. supporting some new variant of markdown?). And someone who already filled match arms for each of the variants can probably also fill in a default match arm for any future evolution :sweat_smile: 

For ReferenceKind, it's kind of the same train of thought, but I'm less sure about myself.

Does my thinking make sense? If so, which ones should I turn exhaustive back?